src/fsharp/AugmentWithHashCompare.fs

```diff
@@ -395,7 +396,7 @@ let mkUnionCompareWithComparer g tcref (tycon:Tycon) (_thisv,thise) (thatv,thate
     let expr = 
         let mbuilder = new MatchBuilder(NoSequencePointAtInvisibleBinding,m ) 
         let mkCase ucase =
-            let cref = mkNestedUnionCaseRef tcref ucase 
+            let cref = tcref.NestedUnionCaseRef ucase 
```

Comment:
what was the `mk`? Shouldn't it be something like `tcref.MakeNestedUnionCaseRef`?